src/components/CardHistory/CardHistory.jsx

```diff
@@ -0,0 +1,100 @@
+import "../../css/main.css";
+import "../CardVideo/CardVideo.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { useHistory } from "../../context";
+
+export const CardHistory = ({ item }) => {
+  const [isPlay, setIsPlay] = useState(false);
+  const [isMoreOptions, setIsMoreOptions] = useState(false);
+  const { _id, title, thumbnail, channel, profile, views, playbackTime } = item;
+
+  const { removeVideoFromHistory } = useHistory();
+
+  useEffect(() => {
+    const clickHandler = () => {
+      if (isMoreOptions) {
+        setIsMoreOptions(false);
+      }
+    };
+    document.addEventListener("click", clickHandler);
```

Comment:
suggestion - we should avoid using document.addEventListener in react. Was it required here ?

Reply:
Yes I tried only using react but could not achieve what I wanted. This is basically to hide the more options div when clicked anywhere on the page or on the more options button again.